python/lsst/pipe/drivers/constructCalibs.py

```diff
@@ -387,7 +387,7 @@ def getOutputId(self, expRefList, calibId):
         outputId.update(calibId)
         return outputId
 
-    def getMjd(self, dataId):
+    def getMjd(self, dataId, timeSystem=dafBase.DateTime.MJD):
```

Comment:
This should be a time scale (`DateTime.UTC` for the old behavior), not a time system. The default value is incorrect in this context (though it may do something because Python treats enums as small ints) and the variable name is misleading.